Mayra/css/styles.css

```diff
@@ -13,15 +13,17 @@ header {
   float: left;
   height: 100%;
 }
-.btn {
+button {
```

Comment:
Careful, this gives all `button`'s on your site the same style. Was that your intent?